README.md

```diff
@@ -49,7 +54,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town.resident.delete(name: "Belle")
+castle.resident.create(name: "Belle")
 ```
```

Comment:
We're working with a normal ruby hash here -- not ActiveRecord. So to delete "Belle" from residents and add her to guests, both of which are arrays, we would so something like this...

```
town["residents"][1].delete
town["castle"]["guests"] << "Belle"
```